app/views/shopify_app/sessions/new.html.erb

```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>Shopify App — Installation</title>
+  <% app_name = ShopifyApp.configuration.app_name %>
+  <title>Shopify App – Installation</title>
```

Comment:
I think you intended to use the variable here? Also you've replaced an em-dash with an ordinary dash. Please use the em-dash.